examples/ex14.cpp

```diff
@@ -104,7 +127,12 @@ int main(int argc, char *argv[])
 
    // 4. Define a finite element space on the mesh. Here we use discontinuous
    //    finite elements of the specified order >= 0.
-   FiniteElementCollection *fec = new DG_FECollection(order, dim);
+   FiniteElementCollection *fec =
+      rk ?
+      (FiniteElementCollection*)new LocalKernelFECollection(dim, rk_num_points,
```

Comment:
Are these casts needed because of the `?:` operator? If that's the case I think `if-else` will look better.

Reply:
Yes, they were. I changed it to use if-else.